bridges/DribbbleBridge.php

```diff
@@ -0,0 +1,33 @@
+<?php
+class DribbbleBridge extends FeedExpander {
+
+	const MAINTAINER = 'quentinus95';
+	const NAME = 'Dribbble popular shots';
+	const URI = 'https://dribbble.com';
+	const CACHE_TIMEOUT = 1800;
+	const DESCRIPTION = 'Returns the newest popular shots from Dribbble.';
+
+	public function collectData(){
+		$html = getSimpleHTMLDOM(self::URI . '/shots')
+			or returnServerError('Error while downloading the website content');
+
+		foreach($html->find('li[id^="screenshot-"]') as $shot) {
+			$item = [];
+			$item['uri'] = self::URI . $shot->find('a', 0)->href;
+			$item['title'] = $shot->find('.dribbble-over strong', 0)->plaintext;
+			$item['author'] = trim($shot->find('.attribution-user a', 0)->plaintext);
+			$item['content'] = $shot->find('.comment', 0)->plaintext;
+			$item['content'] .= $this->getFullSizeImage($shot);
```

Comment:
**Suggestion**:

Use the smaller sized image (suffixed with `_1x` or even smaller `_teaser`) in the content and instead add the full-sized image (no suffix) as `enclosure`. Enclosures will appear for selection in a feed (like an attachment). That way it is possible to quickly browse the contents and manually load full-sized images if interested. You could also encapsulate the `<img` inside an anchor (`<a ...><img ...></a>`) and link the anchor to the full-sized image.

This will also improve rendering time for feeds (especially when viewed in HTML) as the pictures are of smaller size.

See the [Wiki](https://github.com/RSS-Bridge/rss-bridge/wiki/BridgeAbstract#items) for a complete list of supported items

**Hint**: A timestamp would be really neat! :smiley: